lib/github_changelog_generator/generator/generator_fetcher.rb

```diff
@@ -72,7 +72,8 @@ def set_date_from_event(event, issue)
       else
         begin
           commit = @fetcher.fetch_commit(event)
-          issue[:actual_date] = commit[:author][:date]
+          issue[:actual_date] = commit[:commit][:author][:date]
+          # issue[:actual_date] = commit[:author][:date]
         rescue
```

Comment:
One difference in the way the commits are returned with Octokit. This was a nasty bug.